source/administration/changelog.md

```diff
@@ -2,6 +2,168 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
+## Release v3.4.0
+
+Expected release date: 2016-09-16
+
+### Highlights
+
+#### Heroku Deployment
+- Install a preview of Mattermost with one-click using our new Heroku deployment
+
+[![Deploy a Preview](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/mattermost/mattermost-heroku)
+
+#### OAuth 2.0 Service Provider
+- Users with an account on a Mattermost server can securely sign in to third-party application with an OAuth 2.0 protocol. See [documentation](https://docs.mattermost.com/developer/oauth-2-0-applications.html) to learn more.
+
+#### Zapier Integration
+- [Zapier](https://zapier.com) is a service to automate tasks between web apps. You can create zaps that automate posts in Mattermost based on triggers in over [500 supported apps](https://zapier.com/zapbook/) on Zapier. Please [refer to our documentation](https://docs.mattermost.com/integrations/zapier.html) for more details.
```

Comment:
Err, we should change the blog post

![image](https://cloud.githubusercontent.com/assets/14333569/18570374/8e386046-7b66-11e6-87bf-0732608d8f4a.png)